recipe-server/normandy/recipes/models.py

```diff
@@ -330,6 +336,7 @@ def data(self):
             'channels': list(self.channels.all()),
             'countries': list(self.countries.all()),
             'locales': list(self.locales.all()),
+            'identicon_seed': self.identicon_seed,
```

Comment:
@mythmon This means we'll need another signature-busting migration, right?

Reply:
No, because `identicon_seed` isn't a part of the signed part in `canonical_json`.